pkg/controller/podgc/gc_controller.go

```diff
@@ -201,6 +188,32 @@ func (gcc *PodGCController) gcOrphaned(pods []*v1.Pod) {
 	}
 }
 
+func getLatestNodes(nodeListers map[string]clientset.Interface) (map[string]*v1.NodeList, map[string]error) {
```

Comment:
Can we rename nodeListers as kubeClientForNodes as well?

Reply:
renamed to kubeClients